testsuite/integration-arquillian/servers/app-server/tomcat/pom.xml

```diff
@@ -29,18 +29,27 @@
     <packaging>pom</packaging>
     <name>App Server - Tomcat</name>
 
+    <modules>
+        <module>common</module>
```

Comment:
I'm not sure, but it seems you can also skip building the `common` module when all Tomcat profiles are disabled. In Jetty adapter PR, I simply added `common` module to every profile and skipped it with default execution.

Reply:
Good idea. Thanks